handlers/server_client_test.go

```diff
@@ -190,6 +190,7 @@ func TestAcquireByState(t *testing.T) {
 		},
 	}
 	for _, tc := range testcases {
+		tc := tc
```

Comment:
why does lint need this?

Reply:
Because of https://github.com/kyoh86/scopelint/issues/4

Scopelint is deprecated though, we might want to replace it in a followup with exportloopref